healthcheck/healthcheck_test.go

```diff
@@ -2,95 +2,24 @@ package healthcheck
 
 import (
 	"context"
-	"fmt"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
 	"sync"
 	"testing"
 	"time"
 
+	"github.com/containous/traefik/testhelpers"
 	"github.com/vulcand/oxy/roundrobin"
 )
 
 const healthCheckInterval = 100 * time.Millisecond
 
-type testLoadBalancer struct {
```

Comment:
It's a conventional way: when you read test, you beginning by read the test and after you read some test helpers. To read test you don't need to known the test implementation details.